### PR TITLE
Replace /metrics/names with /label/:name/values endpoint.

### PR DESCRIPTION
**Pull request by fabxc**
@juliusv

### CODE REVIEW

File: web/api/v1/api_test.go [hunk 1]
```diff
@@ -110,7 +143,7 @@ func TestEndpoints(t *testing.T) {
 			}
 			continue
 		}
-		if apierr == nil && test.errType != "" {
+		if apierr == nil && test.errType != errorNone {
```
Comment (juliusv) [inline web/api/v1/api_test.go]: `apiErr` seems like the proper camelCasing here.

**Comment by juliusv** [comment]
:+1: modulo nit

File: web/api/v1/api_test.go (hunk 1 above)
Comment (fabxc) [inline web/api/v1/api_test.go]: Done.

**Comment by juliusv** [comment]
:+1:

**Pull request merged**